src/main/java/org/jenkinsci/test/acceptance/controller/TomcatController.java

```diff
@@ -4,11 +4,12 @@
 import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.nio.charset.StandardCharsets;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 
-import org.codehaus.plexus.util.FileUtils;
+import org.apache.commons.io.FileUtils;
```

Comment:
was likely a previous accident, and the `write` taking a `charset` was not present so switched.